libsolc/libsolc.cpp

```diff
@@ -93,9 +93,9 @@ extern char const* solidity_version() noexcept
 {
 	return VersionString.c_str();
 }
-extern char const* solidity_compile(char const* _input, CStyleReadFileCallback _readCallback) noexcept
+extern char const* solidity_compile(char const* _input, CStyleReadFileCallback _readCallback, CStyleReadFileCallback _smtSolver) noexcept
```

Comment:
*sigh* this is a breaking change to the C interface which we wanted to keep stable. What is @axic's opinion on this?

Reply:
@axic seems to be against this, which is also fine with me. IMO it's a feature that would be nice to have but if it has any negative impact we shouldn't go for it.